OpenRA.Mods.Common/Pathfinder/PathSearch.cs

```diff
@@ -120,7 +120,8 @@ public override CPos Expand()
 				var neighborCell = Graph[neighborCPos];
 
 				// Cost is even higher; next direction:
-				if (neighborCell.Status == CellStatus.Closed || gCost >= neighborCell.CostSoFar)
+				if (neighborCell.Status == CellStatus.Closed ||
+					(neighborCell.Status != CellStatus.Unvisited && gCost >= neighborCell.CostSoFar))
```

Comment:
`Cell status is open and ...` is imo easier to read than `cell status is not unvisited and ...`, so I'd use `neighborCell.Status == CellStatus.Open && ...`. What do you think?